plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -196,89 +198,89 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.BIRCH_TRAPDOOR, 0xD4C9A6);
         put(Blocks.BIRCH_WALL_SIGN, 0xC0AF79);
         put(Blocks.BIRCH_WOOD, 0xDBDAD5);
-        put(Blocks.BLACK_BANNER, 0x8F7748);
+        put(Blocks.BLACK_BANNER, 0x000000);
         put(Blocks.BLACK_BED, 0x15151A);
+        put(Blocks.BLACK_CANDLE_CAKE, 0x000000);
         put(Blocks.BLACK_CANDLE, 0x28263C);
-        put(Blocks.BLACK_CANDLE_CAKE, 0x28263C);
         put(Blocks.BLACK_CARPET, 0x15151A);
-        put(Blocks.BLACK_CONCRETE, 0x080A0F);
         put(Blocks.BLACK_CONCRETE_POWDER, 0x191B20);
+        put(Blocks.BLACK_CONCRETE, 0x080A0F);
         put(Blocks.BLACK_GLAZED_TERRACOTTA, 0x451E20);
         put(Blocks.BLACK_SHULKER_BOX, 0x1A1A1E);
+        put(Blocks.BLACK_STAINED_GLASS_PANE, 0x000000);
```

Comment:
I think panels should show up on the map as well. So I just gave them the same color as their full block counterpart.